src/lib/stitching/gravity/stitching.ts

```diff
@@ -137,7 +137,7 @@ export const gravityStitchingEnvironment = (
       }
 
       extend type Viewer {
-        viewingRoomsConnection(first: Int, after: String): ViewingRoomConnection
+        viewingRoomsConnection(first: Int, after: String, statuses: [ViewingRoomStatusEnum!]): ViewingRoomConnection
```

Comment:
Wanted to make sure I understood the syntax of `[ViewingRoomStatusEnum!]`; i.e. what exactly can be null, and found this handy reference: https://graphql-ruby.org/type_definitions/lists.html#nullable-lists-with-non-null-items

TL;DR we can pass `nil` for statuses, but if a list is present, it has to include value(s) of type `ViewingRoomStatusEnum`. Sounds right to me!